apps/publications/src/app/components/publication-filter/publication-filter.component.ts

```diff
@@ -83,8 +85,8 @@ export class PublicationFilterComponent implements OnInit {
     this.code.setValue('');
     this.selectedMode = 'isbn/issn';
     this.selectedCategory = 'no_value';
-    this.startYear = new UntypedFormControl(moment());
-    this.endYear = new UntypedFormControl(moment());
+    this.startYear = new FormControl(moment());
+    this.endYear = new FormControl(moment());
```

Comment:
Can we just set value some default value for the year controls as well ? Instead of creating new objects.